pypdf/_reader.py

```diff
@@ -648,6 +648,7 @@ def _build_field(
                 retval[key][NameObject("/_States_")].append(NameObject("/Off"))
         elif obj.get(FA.FT, "") == "/Btn" and obj.get(FA.Ff, 0) & FA.FfBits.Radio != 0:
             states = []
```

Comment:
```suggestion
            states: List[str] = []
```
should be good/better